templates/form.html

```diff
@@ -52,6 +53,27 @@ <h1 class = "title" >Username Availability Checker</h1>
                 </center>
             </div>
         </div>
+        <div id="usernames">
+            <p class="textualbk"> Need a username? </p>
+            <form onSubmit="generateUsernames(); return false;">
```

Comment:
Moving the `return false` didn't work. I've fixed it by throwing more javascript at the problem. Now using `event.preventDefault` in the function instead of `return false`